fetch-oracle-data.sh

```diff
@@ -1,83 +1,97 @@
 echo "         \"source\": \"coinGecko\""
 echo "       },"
 echo "       {"
-echo "         \"value\": \"$ERGOPOOLADAUSD\","
+echo "         \"value\": \"$(jq '.latest_price' <<< "$ERGO_POOL_ADA_USD_JSON")\","
 echo "         \"source\": \"ergoOracles\""
+echo "       }",
+echo "       {"
+echo "         \"value\": \"$(jq '.ADA.USD' <<< "$CRYPTO_COMPARE_JSON")\","
+echo "         \"source\": \"cryptoCompare\""
 echo "       }"
 echo "    ],"
 echo "    \"ADAEUR\": ["
 echo "       {"
-echo "         \"value\": \"$COINGECKOADAEUR\","
+echo "         \"value\": \"$(jq '.market_data.current_price.eur' <<< "$COIN_GECKO_CARDANO_JSON")\","
 echo "         \"source\": \"coinGecko\""
+echo "       }",
+echo "       {"
+echo "         \"value\": \"$(jq '.ADA.EUR' <<< "$CRYPTO_COMPARE_JSON")\","
+echo "         \"source\": \"cryptoCompare\""
 echo "       }"
 echo "    ],"
 echo "    \"ADABTC\": ["
 echo "       {"
-echo "         \"value\": \"$COINGECKOADABTC\","
+echo "         \"value\": \"$(jq '.market_data.current_price.btc' <<< "$COIN_GECKO_CARDANO_JSON")\","
 echo "         \"source\": \"coinGecko\""
+echo "       },"
+echo "       {"
+echo "         \"value\": \"$(jq '.ADA.BTC' <<< "$CRYPTO_COMPARE_JSON")\","
+echo "         \"source\": \"cryptoCompare\""
 echo "       }"
 echo "    ],"
 echo "    \"BTCUSD\": ["
 echo "       {"
-echo "         \"value\": \"$COINGECKOBTCUSD\","
+echo "         \"value\": \"$(jq '.market_data.current_price.usd' <<< "$COIN_GECKO_BTC_JSON")\","
 echo "         \"source\": \"coinGecko\""
+echo "       }",
+echo "       {"
+echo "         \"value\": \"$(jq '.BTC.USD' <<< "$CRYPTO_COMPARE_JSON")\","
+echo "         \"source\": \"cryptoCompare\""
 echo "       }"
 echo "    ],"
-echo "    \"AGIBTC\": ["
+echo "    \"AGIUSD\": ["
 echo "       {"
-echo "         \"value\": \"$COINGECKOAGIBTC\","
+echo "         \"value\": \"$(jq '.market_data.current_price.usd' <<< "$COIN_GECKO_AGI_JSON")\","
 echo "         \"source\": \"coinGecko\""
+echo "       }",
+echo "       {"
+echo "         \"value\": \"$(jq '.AGI.USD' <<< "$CRYPTO_COMPARE_JSON")\","
+echo "         \"source\": \"cryptoCompare\""
 echo "       }"
 echo "    ],"
 echo "    \"BTCDIFF\": ["
 echo "       {"
-echo "         \"value\": \"$BTCDIFF\","
+echo "         \"value\": \"$(jq -r '.backend.difficulty' <<< "$BTC_TREZOR_JSON")\","
 echo "         \"source\": \"blockBook\""
 echo "       }"
 echo "    ],"
 echo "    \"ERGUSD\": ["
 echo "       {"
-echo "         \"value\": \"$COINGECKOERGUSD\","
+echo "         \"value\": \"$(jq '.market_data.current_price.usd' <<< "$COIN_GECKO_ERGO_JSON")\","
 echo "         \"source\": \"coinGecko\""
 echo "       },"
 echo "       {"
-echo "         \"value\": \"$ERGOPOOLERGUSD\","
+echo "         \"value\": \"$(jq '.latest_price' <<< "$ERGO_POOL_ERG_USD_JSON")\","
 echo "         \"source\": \"ergoOracles\""
 echo "       }"
 echo "    ],"
 echo "    \"TSLA\": ["
```

Comment:
@mmahut I think it should be called: TSLAUSD

Reply:
Is that stock traded in other currency too?

Reply:
I would rather preserve the ticker/name from the sources instead of renaming.

Reply:
I think it is most likely also traded in EUR: https://www.finanzen.net/aktien/tesla-aktie